cfme/v2v/migrations.py

```diff
@@ -0,0 +1,329 @@
+                        ...
+                    }
+        """
+        source_networks_filled = []
+        target_networks_filled = []
+        for cluster in values:
+            self.cluster_selector.fill(cluster)
+            for mapping in values[cluster]['mappings']:
+                source_networks_filled.append(self.source_networks.fill(mapping['sources']))
+                target_networks_filled.append(self.target_networks.fill(mapping['target']))
+                self.add_mapping.click()
+        was_change = any(source_networks_filled) and any(target_networks_filled)
+        if was_change:
+            self.next_btn.click()
+        return was_change
+
+
+class InfraMappingWizardResultsView(View):
+    include_buttons_set1 = View.include(InfraMappingCommonButtonsSet1)
+    next_btn = Button("Close")
+    continue_to_plan_wizard_btn = Button("Continue to the plan wizard")
+
+
+class InfraMappingWizard(View):
+    """Infrastructure Mapping Wizard Modal Widget.
+    Usage:
+        fill: takes values of following format:
+            {
+                'general_view':
+                    {
+                        'name':'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                        'description':fauxfactory.gen_string("alphanumeric",length=50)
+                    },
+                'clusters_view':
+                    {
+                        'mappings': [
+                            {
+                                'sources':['Datacenter \ Cluster'],
+                                'target':['Default \ Default']
+                            }
+                        ]
+                    },
+                'datastores_view':{
+                    'Cluster (Default)': {
+                       'mappings':[
+                            {
+                                'sources':['NFS_Datastore_1','iSCSI_Datastore_1'],
+                                'target':['hosted_storage']
+                            },
+                            {
+                                'sources':['h02-Local_Datastore-8GB', 'h01-Local_Datastore-8GB'],
+                                'target':['env-rhv41-01-nfs-iso']
+                            }
+                        ]
+                   }
+                },
+                'networks_view':{
+                    'Cluster (Default)': {
+                        'mappings': [
+                            {
+                                'sources':['VM Network','VMkernel'],
+                                'target':['ovirtmgmt']
+                            },
+                            {
+                                'sources':['DPortGroup'],
+                                'target':['Storage VLAN 33']
+                            }
+                        ]
+                    }
+                }
+            }
+    """
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    general_view = View.nested(InfraMappingWizardGeneralView)
```

Comment:
Sorry to harp on naming here but I think these can all drop the `_view` suffix and be simply `general`, `clusters`, `datastores`, `networks`, `results`